image.yaml

```diff
@@ -2,36 +2,36 @@ schema_version: 1
 
 name: "redhat-sso-7/sso73"
 description: "Red Hat Single Sign-On 7.3 container image"
-version: "7.3.7.GA"
+version: "7.3.8.GA"
 from: "rhel7:7-released"
 labels:
     - name: "com.redhat.component"
       value: "redhat-sso-7-sso73-container"
     - name: "org.jboss.product"
       value: "sso"
     - name: "org.jboss.product.version"
-      value: "7.3.7.GA"
+      value: "7.3.8.GA"
     - name: "org.jboss.product.sso.version"
-      value: "7.3.7.GA"
+      value: "7.3.8.GA"
 envs:
     - name: "JBOSS_PRODUCT"
       value: "sso"
     - name: "JBOSS_SSO_VERSION"
-      value: "7.3.7.GA"
+      value: "7.3.8.GA"
     - name: "PRODUCT_VERSION"
-      value: "7.3.7.GA"
+      value: "7.3.8.GA"
 
 modules:
       repositories:
           - path: modules
           - name: cct_module
             git:
                 url: https://github.com/jboss-openshift/cct_module.git
-                ref: 0.39.0
+                ref: 0.41.0
```

Comment:
No action required here, just note that this change might or might not work with EAP 7.2.x

Reply:
Tried with local build, the image builds & runs fine:
`12:26:49,621 INFO  [org.jboss.as] (Controller Boot Thread) WFLYSRV0025: Red Hat Single Sign-On 7.3.8.GA (WildFly Core 6.0.26.Final-redhat-00001) started in 14234ms - Started 599 of 858 services (560 services are lazy, passive or on-demand)
`
We will see how it behaves directly on OpenShift though. Resolving for now (will downgrade back to `0.39.0` in case of issues).